source/extensions/filters/listener/http_inspector/http_inspector.h

```diff
@@ -2,6 +2,7 @@
 
 #include "envoy/event/file_event.h"
 #include "envoy/event/timer.h"
+#include "envoy/extensions/filters/listener/http_inspector/v3/http_inspector.pb.h"
```

Comment:
Is this needed?

Reply:
Ah, no, just missed removing it with the configurability removal.